.travis.yml

```diff
@@ -104,13 +104,6 @@ jobs:
               - npm run lint:ci
           env:
               - WOOCOMMERCE_BLOCKS_PHASE=3
-        - name: E2E Tests (WP 5.2)
-          script:
-              - npm run test:e2e
-          env:
-              - WP_VERSION=5.2
-              - E2E_TESTS=1
-              - WOOCOMMERCE_BLOCKS_PHASE=3
```

Comment:
Deleting this is only half of it, previously we supported `5.2`, `5.3`, `5.4`, and `5.5 + GB` now we should add `5.5` and `5.5 + GB`, which means you should leave everything as is and just update the `WP_VERSION` var and name in each suite.

Reply:
Thanks @senadir , that makes sense 😄